agenta-cli/agenta/config.toml

```diff
@@ -2,4 +2,6 @@ docker_registry_url="127.0.0.1:5001"
 database_url="localhost:5432"
 registry="agenta-server"
 backend_url_suffix="api"
+tracking_enabled=true
```

Comment:
no the right config.toml see other comment

Reply:
I get this:
![image](https://github.com/Agenta-AI/agenta/assets/55067204/64df24e3-83cd-4bed-bcb7-7053672e7c67)